src/Plugin/Patches.php

```diff
@@ -85,23 +85,6 @@ class Patches implements PluginInterface, EventSubscriberInterface, Capable
 
     protected JsonFile $lockFile;
 
-    /**
-     * Get the path to the current patches lock file.
-     */
-    public static function getPatchesLockFilePath(): string
```

Comment:
I can't remember why this was a static function. Might be worth checking if this breaks any of the tests.

Reply:
Ah, yeah. https://github.com/cweagans/composer-patches/actions/runs/9645839515/job/26616352752?pr=584#step:5:137

Reply:
By not calling the method while registering, its fixed.
Parsing extra's just for the description is not worth it?